code/script.js

```diff
@@ -1,325 +1,539 @@
+  // We don't want to show the whole time in second, only the one less than 60
+  const sec = totalSecond % 60;
 
-// Draw the game board
+  // Divide the total amount to 60 to get the minute. Use Math.floor to ground the number
+  const min = Math.floor((totalSecond/60));
+
+  // Set up how to display second
+  if (sec < 10) {
+    second.textContent = `0${sec}`
+  } else  {
+  second.textContent = sec
+  }
+  
+  // Set up how to display minute
+  if (min < 10) {
+    minute.textContent = `0${min} : `
+  } else {
+    minute.textContent = `${min} : `
+  }
+}
+
+// Invoke the function after every 1 second
+
+setInterval(countTime,1000)
+
+
+// 3. Draw the game board
 const generateBoard = () => {
   board.innerHTML = ''
   charactersInPlay.forEach((person) => {
     board.innerHTML += `
-      <div class="card">
+      <div class="card" onmouseover="playSound()">
         <p>${person.name}</p>
         <img src=${person.img} alt=${person.name}>
         <div class="guess">
           <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          <button class="filled-button small guess-btn" onclick="guess('${person.name}')" >Guess</button>
         </div>
       </div>
     `
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// 3. Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
 }
 
-// This function to start (and restart) the game
+
+// 4. This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
+  charactersInPlay = CHARACTERS;
+
   // What else should happen when we start the game?
+  winOrLose.style.display = "none"
+  board.style.display = "flex"
+
+
+  // To load the game interface
+  generateBoard();
+
+  // Get secret person
+  setSecret()
+
+  // To reset the time
+  totalSecond =0;
+
+  // To reset the number of guess an display it
+  numberOfGuess = 0;
+  guessMade.textContent = numberOfGuess;
+
+  // To hide the greeting modal 
+  inputWrapper.style.display = "none"
 }
 
-// setting the currentQuestion object when you select something in the dropdown
+
+
+// 5. setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
 
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
+   const value = questions.value;
```

Comment:
Didn't know you could type it out this short, neat! (I copied the whole thing from 'const category' and changed the last part to .value)